docs/concepts/filesystems.md

```diff
@@ -185,6 +185,18 @@ You need to install `adlfs` to use it.
 If you leverage `S3`, `GCS`, or `Azure` storage blocks, and you don't explicitly configure credentials on the respective storage block, those credentials will be inferred from the environment. Make sure to set those either explicitly on the block or as environment variables, configuration files, or IAM roles within both the build and runtime environment for your deployments.
 
 
+## Handling file-system-specific package dependencies in Docker images
```

Comment:
```suggestion
## Filesystem-specific package dependencies in Docker images
```

Reply:
Suggested change to limit wrapping in the left nav. But trying to not change the meaning too much. Ignore if this adds confusion.